### DISCUSSION TITLE
How to Emulate Boolean Flags

### DISCUSSION THREAD

**Discussion by hisaac** [casey/just#983] [Q&A]
One feature I wish `just` had is the ability to pass boolean flags to commands. Something like:

```shell
just clean --deep
```

So adding `--deep` to the clean command will do a deep clean, whereas omitting it will do just a regular clean.

I'm new to the tool, but as far as I can tell, there's no native/official way to do this — please tell me if I'm wrong though. Barring an official way, has anyone worked up something that works similarly to boolean flags with their `just` setup?

**Comment by casey** [comment]
There's actually an issue for this, but nobody's tackled it yet: #476.

**Reply by hisaac** [reply]
Thanks @casey, I went looking, but didn't see that one. I'll subscribe to the issue to watch its progress.
